hwpf/fapi2/src/plat/plat_error.C

```diff
@@ -49,6 +51,79 @@ bool clock_redundancy_is_enabled()
 	return false;
 }
 
+/**
+ * @brief Helper function to update FFDC to log informational logs
+ *        for the redundant mode enabled systems
+ *
+ * The defined api is used to modify the FFDC information with custom one
+ * to handle spare clock failure. Steps involved
+ * 	- Remove all CDG records from the input FFDC structure
+ * 	- Update custom CDG record, only deconfigure failing clock unit
+ * 	- Update FFDC type to FFDC_TYPE_CLOCK_INFO, to help api users
+ * 	  to log informationla logs.
+ *
+ * @param[in,out] ffdc  ffdc data which need to be updated if it is clock
+ *                      failure
+ * @param[in] hwCallout   reference of hwcallout with clock related hwid
+ *
+ * @return void
+ */
+void clock_callout_info(FFDC &ffdc, HWCallout &hwCallout)
+{
+	// Get oscrefclock pdbg target based clock position
+	auto clk_pos = hwCallout.clkPos;
+	struct pdbg_target *clock_target;
+	uint8_t attr_clk_pos = 0;
+	pdbg_for_each_class_target("oscrefclk", clock_target)
+	{
+		if (!pdbg_target_get_attribute(clock_target, "ATTR_POSITION", 2,
+					       1, &attr_clk_pos)) {
+			FAPI_ERR("Attribute ATTR_POSITION read failed"
+				 " for clock '%s' \n",
+				 pdbg_target_path(clock_target));
+			return;
+		}
+		if (attr_clk_pos == clk_pos)
```

Comment:
no need of braces {} also may be need to run clang on this file

Reply:
since it is one liner skipped {{}
clang already ran here.